File: src/ca/qc/bdeb/inf203/model/combatants/ExplosionNucleaire.java
package ca.qc.bdeb.inf203.model.combatants;

import ca.qc.bdeb.inf203.model.Entite;
import ca.qc.bdeb.inf203.model.Etat;

/**
 * Explosion nucléaire qui tue tout.
 *
 * @author Nicolas Hurtubise
 */
public class ExplosionNucleaire extends Entite {

    @Override
    public int getAnimationCompteur() {
        return 0;
    }

    @Override
    public Etat getEtat() {
        return Etat.ATTAQUE;
    }
}

File: assets/js/model/combatants/explosion-nucleaire.js
/**
 * Explosion nucléaire qui tue tout.
 *
 * @author Nicolas Hurtubise
 */
function ExplosionNucleaire() {
    Entite.apply(this);
}

ExplosionNucleaire.prototype = new Entite();

ExplosionNucleaire.prototype.getAnimationCompteur = function() {
    return 0;
};

ExplosionNucleaire.prototype.getEtat = function() {
    return navigator.Etat.ATTAQUE;
};
